### ISSUE TITLE
[ERROR]: The similarity metrics table is not clear.

### ISSUE THREAD

**Issue by JackLCL** [milvus-io/milvus-docs#1492]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Issue

link: https://milvus.io/docs/v2.0.x/metric.md#floating

![image](https://user-images.githubusercontent.com/53512883/165008707-c4ab3f1c-f2ca-41c4-880f-b8cd897802f3.png)

All current index types support L2, but this table shows that only flat indexes support L2.

### Suggestion

It is recommended to redesign the table to express the meaning more clearly.

### Anything else?

N/A

**Comment by claireyuw** [comment]
@JackLCL The table layout lists all the index types supported by both L2 and IP, although it could use some improvements. Thanks for pointing out the confusion!
Maybe remake the table to format it like this? @LocoRichard what do you think?
<img width="818" alt="image" src="https://user-images.githubusercontent.com/83751381/165015263-63478a18-eb97-49e6-b954-5c100114094d.png">

**Comment by LocoRichard** [comment]
> @JackLCL The table layout lists all the index types supported by both L2 and IP, although it could use some improvements. Thanks for pointing out the confusion! Maybe remake the table to format it like this? @LocoRichard what do you think? <img alt="image" width="818" src="https://user-images.githubusercontent.com/83751381/165015263-63478a18-eb97-49e6-b954-5c100114094d.png">

Not really. First, the grid would be meaningless if all boxes are ticked. Second, it would lead to too many columns in a table and would be very unfriendly for rendering the page layout. I think the original one is ok. If we want the items correspond to each other, I would put them in different rows.
